src/java.base/share/classes/java/security/Key.java

```diff
@@ -86,6 +86,10 @@
  * Security Appendix</a>
  * of the <cite>Java Object Serialization Specification</cite> for more information.
  *
+ * @spec serialization/index.html Java Object Serialization Specification
```

Comment:
How about "Java Security Standard Algorithm Names Specification"? Should its reference be covered by @spec tag?